platform-sdk/swirlds-virtualmap/src/test/java/com/swirlds/virtualmap/internal/merkle/VirtualInternalNodeTest.java

```diff
@@ -159,6 +163,8 @@ void getValidChildOnDisk() throws IOException {
         final VirtualRootNode<TestKey, TestValue> root = createRoot();
         root.getState().setFirstLeafPath(6);
         root.getState().setLastLeafPath(12);
+        final KeySerializer<TestKey> keySerializer = new TestKeySerializer();
```

Comment:
`TestKeySerializer` and `TestValueSerializer` are stateless. Can you make them singletons and use these singletons instead of creating instances in multiple tests?

Reply:
Makes sense. I introduced `TestKeySerializer.INSTANCE` and `TestValueSerializer.INSTANCE` static final fields and used them all over the tests.